README.md

```diff
@@ -170,13 +168,13 @@ Other Style Guides
       id: 5,
       name: 'San Francisco',
     };
-    obj[getKey('enabled')] = true;
+    obj[getKey("enabled")] = true;
```

Comment:
It's going to be fun changing single to double quotes every time there are upstream changes to review and merge... :-/